index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
```

Comment:
nice and clean html!
lets use double space instead of tabs
let me know if you need help with setup